blobstore/cli/clustermgr/list_all.go

```diff
@@ -188,6 +200,21 @@ func listAllVolumes(volumeTbl *volumedb.VolumeTable) error {
 	})
 }
 
+func listAllVolumeUnits(volumeTbl *volumedb.VolumeTable) error {
+       return volumeTbl.RangeVolumeUnits(func(unitRecord *volumedb.VolumeUnitRecord) error {
+               unitInfo := unitInfo{
+                       VolumeUnitRecord: *unitRecord,
+               }
+
+               data, err := json.Marshal(unitInfo)
+               if err != nil {
+                       return fmt.Errorf("json marshal failed, err: %s", err.Error())
```

Comment:
panic is better.  There's no need to change the argument of volumedb's RangeVolumeUnits

Reply:
correct-code: `func (v *VolumeTable) RangeVolumeUnits(f func(unitRecord *VolumeUnitRecord) error) (err error)`

error-code: `func (v *VolumeTable) RangeVolumeUnits(f func(unitRecord *VolumeUnitRecord)) (err error)`

compile ERROR:
```
build blobstore    # github.com/cubefs/cubefs/blobstore/cli/clustermgr
blobstore/cli/clustermgr/list_all.go:202:36: cannot use func literal (type func(*volumedb.VolumeUnitRecord) error) as type func(*volumedb.VolumeUnitRecord) in argument to volumeTbl.RangeVolumeUnits
failed
```